Tests/SwiftSyntaxHighlighterTests/CommandTests.swift

```diff
@@ -28,51 +33,60 @@ final class CommandTests: XCTestCase {
         }
         process.standardOutput = outputPipe
 
+        process.standardError = FileHandle.nullDevice
```

Comment:
Oh huh, I don't think I've seen `nullDevice` before. I think I'll adopt this pattern the next time I do I/O in Swift.